.travis/apisix_cli_test.sh

```diff
@@ -389,12 +389,86 @@ if [ ! $? -eq 0 ]; then
     exit 1
 fi
 
-git checkout conf/config.yaml
+echo "passed: access_log_format in nginx.conf is ok"
 
-echo "passed: worker_processes number is configurable"
+# check enable access log
+
+echo '
+nginx_config:
+  http:
+    enable_access_log: true
+    access_log_format: "$remote_addr - $remote_user [$time_local] $http_host test_enable_access_log"
+' > conf/config.yaml
+
+make init
+
+grep "test_enable_access_log" conf/nginx.conf > /dev/null
+if [ ! $? -eq 0 ]; then
+    echo "failed: nginx.conf file doesn't find access_log_format when enable access log"
+    exit 1
+fi
+
+count_access_log_off=`grep -c "access_log off;" conf/nginx.conf || true`
+if [ $count_access_log_off -eq 2 ]; then
+    echo "failed: nginx.conf file find access_log off; when enable access log"
+    exit 1
+fi
+
+make run
+sleep 0.1
+curl http://127.0.0.1:9080/hi
+sleep 4
+tail -n 1 logs/access.log > output.log
+
+grep "test_enable_access_log" output.log > /dev/null
+if [ ! $? -eq 0 ]; then
+    echo "failed: not found test_enable_access_log in access.log "
+    exit 1
+fi
+
+make stop
+
+echo '
+nginx_config:
+  http:
+    enable_access_log: false
+    access_log_format: "$remote_addr - $remote_user [$time_local] $http_host test_enable_access_log"
+' > conf/config.yaml
+
+make init
+
+grep "test_enable_access_log" conf/nginx.conf > /dev/null
```

Comment:
Use `grep -c xx || true` for this line?

Reply:
Do you know the reason why this test case is failed? It is a little bit confusing~

Reply:
Because there is `set -e` at the top of the file.

Reply:
Oh, I see. Thank you:smile: